sno/git_util.py

```diff
@@ -6,6 +6,38 @@
 from .timestamps import tz_offset_to_minutes
 
 
+def get_head_tree(repo):
+    """Returns the tree at the current repo HEAD."""
```

Comment:
Can the docstring be expanded with the comments about when/why `None` is returned?

Reply:
Done. Also fixed the try/except pattern - we should have just been checking if head_is_unborn